GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -1079,6 +1100,24 @@ private void RevisionGridSelectionChanged(object sender, EventArgs e)
             }
         }
 
+        private async Task RevisionGridSelectionChangedAsync(object sender, EventArgs e)
+        {
+            try
+            {
+                _selectedRevisionUpdatedTargets = UpdateTargets.None;
+
+                FillFileTree();
+                FillDiff();
+                await FillCommitInfoAsync();
```

Comment:
❗️ Missing `ConfigureAwait(true)`

⚠️ This comment is different from most others. Since the operation after the `await` appears to be a UI refresh operation, it passes an explicit `true` to `ConfigureAwait`.